pages/security/device-security/software-safety/pt.md

```diff
@@ -0,0 +1,64 @@
+@title = "Boas Práticas de Segurança de Software"
+@nav_title = "Segurança de Software"
+@summary = "Boas práticas para instalar e atualizar software"
+@toc = true
+
+# Mantenha seu software atualizado
+
+Quase toda vez que um atacante consegue compremeter um computador através de uma vulnerabilidade no software, o ataque poderia ter sido prevenido se o software estivesse atualizado. Isso é porque quase todas as vulnerabilidades são conhecidas pelos desenvolvedores do software e já foram consertadas no momento em que o atacante aprende a se aproveitar das falhas.
+
+É especialmente importante manter seu sistema operacional atualizado, já que o sistema operacional tem acesso privilegiado a tudo que acontece em seu dispositivo.
+
+Como ativar atualizações automáticas:
+
+* **macOS**: A partir do menu **Apple**, selecione **Preferências do Sistema** > **App Store** > **Baixar novas atualizações quando disponíveis**.
+* **Windows**: Clique no botão **Iniciar**, selecione **Configurações** > **Atualização e Segurança** > **Windows Update** > **Opções avançadas** e em **Escolha como as atualizações serão instaladas**, selecione **Automático (recomendado)**.
+
+Observações:
+
+* Você pode precisar reiniciar seu dispositivo para que diversas atualizações sejam aplicadas, então permitir que seu dispositivo reinicie automaticamente após uma atualização é necessário para que a atualização possa prover proteção.
+* Se você tem requisitos específicas para softwares ou softwares customizados para sua organização, atualizações automáticas podem causar interrupções em seu trabalho, já que algumas atualizações do sistema operacionakl podem ser incompatíveis com os softwares existentes.
+* Manter seu software atualizado não ajuda com ataques de Phishing ou Malware, que funcionam enganando o usuário a realizar certo comportamento inseguro.
+
+# Use a loja de aplicações
+
+Quando possível, é melhor instalar software pela loja de aplicações oficial do seu sistema operacional, por várias razões:
+
+* **Autenticidade**: Aplicações na loja de aplicações oficial são assinadas pelo desenvolvedor e verificada pela loja de aplicações.
+* **Atualizações**: A loja de aplicações vai te ajudar a manter suas aplicações atualizadas.
+* **Prevenção de Ataques de Side Loading**: Instalar uma aplicação por fora da loja oficial de aplicações é chamado de "side loading". Side loading é atualmente desabilitado por padrão no Android, impossível no iOS, e é is habilitado por padrão no macOS e Windows. Quando o side loading está habilitado, seu computador fica mais vulnerável a diferentes formas de ataque de malware. Por essa razão, ambos macOS e Windows estão tentando migrar para um modelo em que side loading esteja desabilitado por padrão.
```

Comment:
- [ ] Sugestão mudar "aplicações" para "aplicativos".
- [ ] Remover palavra extra em inglês: "impossível no iOS, e é **is** habilitado por padrão"